components/eam/bld/config_files/definition.xml

```diff
@@ -44,7 +44,7 @@ Option to turn on waccmx thermosphere/ionosphere extension: 0 => no, 1 => yes
 <entry id="phys" valid_values="cam3,cam3_5_1,cam4,cam5,ideal,adiabatic,default" value="">
 Physics package: cam3, cam4, cam5, ideal (Held-Suarez forcings), adiabatic, default (eam).
 </entry>
-<entry id="microphys" valid_values="rk,mg1,mg1.5,mg2" value="">
+<entry id="microphys" valid_values="rk,mg1,mg1.5,mg2,none" value="">
 Microphysics package: rk (Rasch and Kristjansson), mg1 (Morrison and Gettelman two moment scheme CAM5.1), mg1.5 (Morrison and Gettelman second version development), mg2 ((Morrison and Gettelman second version).
```

Comment:
Maybe add a note here specifying that "none" means do not run microphysics? Maybe that's obvious but I could see confusion about whether "none" means "select a default" or just "do not run microsphysics at all".

Reply:
good point